docs/de/about/README.md

```diff
@@ -0,0 +1,14 @@
+---
+home: false
+# layout: BlogHome
+icon: home
+title: Über
+sidebar: false
+tagLine: 
+# actions:
+#   - text: Demo
+#     link: https://stage.ocelot.social/
+---
+## XXX
```

Comment:
Same here:

- As loang as we do not have content here, [this page](/de/contact/) should not be available

Reply:
See above

Reply:
Done